pip/templates/deploy.py

```diff
@@ -29,54 +29,44 @@
 # noinspection PyUnresolvedReferences
 import twine.commands.upload
 
-parser = argparse.ArgumentParser()
-parser.add_argument('repo_type')
-args = parser.parse_args()
+pypi_profile = "{pypi_profile}"
+pip_registry = "{snapshot}" if "{snapshot}" else "{release}"
+command = None
 
-repo_type_key = args.repo_type
-
-pip_repositories = {
-    'snapshot' : "{snapshot}",
-    'release' : "{release}"
-}
-
-pip_registry = pip_repositories[repo_type_key]
-
-pip_username, pip_password = (
-    os.getenv('DEPLOY_PIP_USERNAME'),
-    os.getenv('DEPLOY_PIP_PASSWORD'),
-)
-
-if not pip_username:
-    raise Exception(
-        'username should be passed via '
-        'DEPLOY_PIP_USERNAME env variable'
+if pypi_profile:
+    command = ['./dist/*', '--repository', pypi_profile]
+else:
+    pip_username, pip_password = (
+        os.getenv('DEPLOY_PIP_USERNAME'),
+        os.getenv('DEPLOY_PIP_PASSWORD'),
     )
 
-if not pip_password:
-    raise Exception(
-        'password should be passed via '
-        '$DEPLOY_PIP_PASSWORD env variable'
-    )
+    if not pip_username:
+        raise Exception(
+            'username should be passed via '
+            'DEPLOY_PIP_USERNAME env variable'
+        )
+
+    if not pip_password:
+        raise Exception(
+            'password should be passed via '
+            '$DEPLOY_PIP_PASSWORD env variable'
+        )
+    command = ['./dist/*', '-u', pip_username, '-p', pip_password, '--repository-url', pip_registry]
 
 with open("{version_file}") as version_file:
     version = version_file.read().strip()
-
 new_package_file = None
-
+new_wheel_file = None
 try:
-    new_package_file = "{package_file}".replace(".tar.gz", "-{}.tar.gz".format(version))
+    os.mkdir("./dist")
+    new_package_file = "./dist/" + "{package_file}".replace(".tar.gz", "-{}.tar.gz".format(version))
+    new_wheel_file = "./dist/" + "{wheel_file}".replace(".whl", "-{}.whl".format(version))
+
     shutil.copy("{package_file}", new_package_file)
+    shutil.copy("{wheel_file}", new_wheel_file)
 
-    twine.commands.upload.main([
-        new_package_file,
-        '-u',
-        pip_username,
-        '-p',
-        pip_password,
-        '--repository-url',
-        pip_registry
-    ])
+    twine.commands.upload.main(command)
 finally:
     if new_package_file:
         os.remove(new_package_file)
```

Comment:
We should probably remove the `new_wheel_file` here as well.

Reply:
I have added a new commit to address this issue.